data_steward/tools/run_cleaning_rules.sh

```diff
@@ -49,6 +49,13 @@ if [[ -z "${key_file}" ]] || [[ -z "${app_id}" ]] || [[ -z "${vocab_dataset}" ]]
   exit 1
 fi
 
+ROOT_DIR=$(git rev-parse --show-toplevel)
```

Comment:
This file still has the redundant app_id variable. Please add TODO if deferring absolute path resolution fix.